homeassistant/components/motion_blinds/__init__.py

```diff
@@ -164,7 +164,7 @@ def stop_motion_multicast(event):
     device_registry.async_get_or_create(
         config_entry_id=entry.entry_id,
         connections={(dr.CONNECTION_NETWORK_MAC, motion_gateway.mac)},
-        identifiers={(DOMAIN, entry.unique_id)},
+        identifiers={(DOMAIN, motion_gateway.mac)},
```

Comment:
Do we need to migrate the existing entries to avoid orphaning the old one?

Reply:
See https://github.com/home-assistant/core/pull/64698#discussion_r790137539 they were originally created using gateway mac, just not grouped with it.